backend/routers/userRouter.js

```diff
@@ -17,7 +18,8 @@ userRouter.post('/authenticate', authenticate);
 userRouter.put('/update-user-info', verifyToken, updateUserInfo);
 userRouter.post('/assign-role', verifyAdmin, assignRole);
 userRouter.post('/assign-manager', verifyAdmin, assignManager);
-userRouter.get('/get-users', verifyAdmin, getUsers);
+userRouter.get('/get-users', getUsers);
 userRouter.post('/create-personalDetails', createPersonalDetails);
+userRouter.post('/create-employeeDetails', createEmployeeDetails);
```

Comment:
If model name is EmploymentDetails won't it be better to call `create-employeeDetails` to `create-employmentDetails`